kickstarter/android-oss
# NTV-273: Location Model

# 📲 What
- Migrated to kotlin
- Adopt @Parcelize 

# 🤔 Why
- Remove all references to AutoParcel library in Models, eventually we will be able to remove that dependency.

# 👀 See
- No user facing changes

|  |  |

# 📋 QA
- Check any of your previously backed projects, update the pledge by changing the location.
- Pledge to a project
- Navigate over a few projects

# Story 📖

[NTV-273](https://kickstarter.atlassian.net/browse/NTV-273)
